.github/workflows/upgrade-python-requirements.yml

```
@@ -14,6 +14,7 @@ jobs:
     with:
        branch: ${{ github.event.inputs.branch }}
        send_success_notification: false
+       team_reviewers: "arbi-bom"
```

```suggestion
       team_reviewers: "2u-arbi-bom"
```
This should match the github team name.